src/components/Popup.js

```diff
@@ -1,34 +1,32 @@
 export default class Popup {
-  constructor({ popupSelector }) {
+  constructor(popupSelector) {
     this._popupElement = document.querySelector(popupSelector);
-    this._popupElementCloseBtn =
-      this._popupElement.querySelector(".modal__close");
+    this._closeByEscape = this._closeByEscape.bind(this);
   }
 
   open() {
     this._popupElement.classList.add("modal_opened");
-    document.addEventListener("keydown", this._handleEscClose);
+    document.addEventListener("keydown", this._closeByEscape);
   }
 
   close() {
     this._popupElement.classList.remove("modal_opened");
-    document.removeEventListener("keydown", this._handleEscClose);
+    document.removeEventListener("keydown", this._closeByEscape);
+    this._popupElement.removeEventListener("mousedown", this._closeOverlaybyClick);
```

Comment:
`    this._popupElement.removeEventListener("mousedown", this._closeOverlaybyClick);
`

Please, delete it

You don't need it here because you don't add such a listener